build/build-release.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+set -euo pipefail
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+BOOTKUBE_ROOT=${BOOTKUBE_ROOT:-${SCRIPT_DIR}/..}
+BOOTKUBE_RELEASE=${BOOTKUBE_BIN:-${BOOTKUBE_ROOT}/_output/release/bootkube-linux-amd64.tar.gz}
+DOCKER_REPO=${DOCKER_REPO:-quay.io/coreos/bootkube}
+DOCKER_TAG=${DOCKER_TAG:-$(${BOOTKUBE_ROOT}/build/git-version.sh | tr + _)}
```

Comment:
Right now it's not strictly necessary, because we have no tags. But tags that contain semver comments (e.g. `+foo.1`) would break the docker tags. Mostly a habit from hyperkube releases because there are always comment tags.

I'll remove for now to keep complexity lower, and if we need it later it's easy to add back.